### PR TITLE
EWL-4896 input atom

### PR DESCRIPTION
**Pull request by designerneil**
## Ticket(s)

**Jira Ticket**
- [EWL-4896: SG2 | Create "Date field" atom](https://issues.ama-assn.org/browse/EWL-4896)

## Description
Create input field atom.

## To Test
- [ ] `gulp serve`
- [ ] visit http://localhost:3000/?p=atoms-text-field
- [ ] Observe label that says LABEL
- [ ] Observe an input field with "Text field placeholder" for a placeholder
- [ ] Observe the help text below the input

## Visual Regressions
backstop test only noted the footer organism as failures due to it shifting positions.

## Relevant Screenshots/GIFs
screen shot 2018-04-24 at 11 20 53 am

## Remaining Tasks
N/A

## Additional Notes
This PR was branched off the date field PR https://github.com/AmericanMedicalAssociation/ama-style-guide-2/pull/191. You will notice files being deleted in this PR. This discrepancy will go away once the date-field PR is merged and this PR is updated after.
---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

### CODE REVIEW

File: styleguide/source/_patterns/01-atoms/forms/date-field.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (labbydev) [inline styleguide/source/_patterns/01-atoms/forms/date-field.json:1]: This should likely not be included in this PR.

**Review by labbydev — COMMENTED** [review]
Please review the comments from PR #191 that would have the same comments here. 

Also looking at the zeplin it looks like the help text may need to be closer to the input. We may want to confirm the spacing and sizing on these with UX.

File: styleguide/source/_patterns/01-atoms/forms/date-field.json (hunk 1 above)
Comment (designerneil) [reply styleguide/source/_patterns/01-atoms/forms/date-field.json:1]: @labbydev I already asked @toph and he said the default spacing should be fine

Comment (designerneil) [reply styleguide/source/_patterns/01-atoms/forms/date-field.json:1]: @labbydev this PR helps the look and feel for the telephone PR and should be merged into develop before https://github.com/AmericanMedicalAssociation/ama-style-guide-2/pull/192

**Review by labbydev — APPROVED** [review]
